src/main/java/com/qunite/api/security/JwtConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.qunite.api.security;
+
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class JwtConstants {
+  public static final String TOKEN_PREFIX = "Bearer ";
```

Comment:
This is a prefix only for the token bearer, I don't see the point of putting it in a constant